volume/store/store_test.go

```diff
@@ -5,11 +5,15 @@ import (
 	"strings"
 	"testing"
 
+	pluginstore "github.com/docker/docker/plugin/store"
 	"github.com/docker/docker/volume/drivers"
 	volumetestutils "github.com/docker/docker/volume/testutils"
 )
 
 func TestCreate(t *testing.T) {
+	pluginStore := pluginstore.NewStore("/var/lib/docker")
```

Comment:
It's the rootdir where plugin rootfs and other meta data are created. Its the same as dockerd's graphRoot.